internal/cfgplugins/staticnexthop.go

```diff
@@ -171,6 +172,20 @@ type StaticNextHopGroupParams struct {
 
 }
 
+// NexthopGroupUDPParams defines the parameters used to create or configure a Next Hop Group that performs UDP encapsulation for traffic forwarding.
+type NexthopGroupUDPParams struct {
+	TrafficType        string
```

Comment:
Add comment explaining purpose of this var.  Looks like it is used for IP address family?  Maybe rename it so it is more specific